### ISSUE TITLE
Avoid oversized avatar image

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#10369]
## Bug description 

Sometimes users fail to find the menu, where they can check their gc.com credentials and/or reauthorize although reading our FAQ and/or user guide.  Last days I had contact to a user who also failed to find it, so I asked for screenshots to guide him through. 
It turned out his avatar picture was blocking the relevant menu (see screenshot below).

Now, that I have seen this, it explains quite some support tickets, where people had problems to follow, what I instructed them to do.

### Steps to reproduce the problem

- Open c:geo Menu - Settings - Services
- Click on geocaching.com

### Actual result after these steps 

For some users the avatar image in that menu is just oversized blocking out the text next to it

### Expected result after these steps 

We should enforce a certain size of that image (in percentage of screen), so that it always looks as defined/desired

## c:geo version 

2021.02.14

## Reproducible 

No, I tried uploading several new avatar images on gc.com, but the scale in our menu was always OK.

## System information 

```
--- System information ---
Device: SM-G975U (beyond2qltesq, samsung)
Android version: 11
Android build: RP1A.200720.012.G975USQU5GUBH
c:geo version: 2021.02.14
Google Play services: disabled - 21.09.15 (150400-361652764)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide caches: -
Hide waypoints: -
HW acceleration: enabled (default state)
System language: en_US / user-defined language:
System date format: M/d/yy
Debug mode active: no
System internal c:geo dir: /data/user/0/cgeo.geocaching (33.5 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (33.4 GB free) external non-removable
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (33.4 GB free) external non-removable
Database: /data/user/0/cgeo.geocaching/databases/data (2.1 MB) on system internal storage
Last backup: never
GPX import path: /storage/emulated/0/cgeo/gpx
GPX export path: /storage/emulated/0/cgeo/gpx
Offline maps path: null
Map render theme path:
Live map mode: true
Global filter: display all caches
Sailfish OS detected: false
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Not logged in (Login failed)
Geocaching.com date format: MM/dd/yyyy
Installed c:geo plugins: contacts
BRouter connection available: false
--- End of system information ---
```

## Screenshots 

Anonymized screenshot
![avatar_problem](https://user-images.githubusercontent.com/949669/114302493-98f18080-9ac9-11eb-9c78-5f242735d275.jpg)

**Comment by moving-bits** [comment]
Can't we simply remove the avatar image?
AFAIK it's only used in the login settings page, so not much to lose, IMHO.

**Comment by Lineflyer** [comment]
Yes, we could.
It was initially introduced as a kind of confirmation / visual feedback, that they logged in to the right account.

Isn't there an eesy possibility to define a max size?

**Comment by moving-bits** [comment]
> Isn't there an eesy possibility to define a max size?

Not that I know of. I remember that we already have an issue somewhere regarding the avatar blocking the re-auth function somewhere, and that we tried back then to somehow force the image "to stay in line", but did not find an easy way to do so, due to the somewhat strange way displaying the image is done technically.

**Comment by Lineflyer** [comment]
My opinion then: Remove it if it makes our menu look unpredictable.

**Comment by moving-bits** [comment]
fixed with #10390 by restricting available space for the avatar image
